www/hockeyapp.js

```diff
@@ -20,6 +20,9 @@ var hockeyapp = {
     feedback: function (success, failure) {
         exec(success, failure, "HockeyApp", "feedback", []);
     },
+    feedbackModal: function (success, failure, takeScreenshot, data) {
```

Comment:
What do you think about calling the parameter `attachScreenshot`, that way it's a little clearer that the screenshot is being attached to the feedback?